admin/templates/spam/spam.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
```

Comment:
This file might make more sense named `spam/spam_list.html`. The naming convention in other template directories is that `<model_name>.html` is the view for a `<model>` instance, whereas this is the view for a list of spam.